.github/workflows/auth-test.yml

```diff
@@ -0,0 +1,37 @@
+name: 'Test gcloud authentication'
+
+on:
+  pull-request:
```

Comment:
Let's change this back to push now that we verified it's working.

Reply:
Deleted this check action